website/source/assets/stylesheets/_inner.scss

```diff
@@ -1,11 +1,31 @@
 #inner {
-  p, li, .alert {
-    font-size: $font-size;
-    font-family: $font-family-open-sans;
-    font-weight: $font-weight-reg;
-    line-height: 1.84em;
-    margin: 0 0 $font-size;
-    -webkit-font-smoothing: antialiased;
+  h1 {
+    @extend .g-type-display-2;
+  }
+
+  h2 {
+    @extend .g-type-display-3;
+  }
+
+  h3 {
+    @extend .g-type-display-4;
+  }
+
+  h4 {
+    @extend .g-type-display-5;
+  }
+
+  h5,
+  h6 {
+    @extend .g-type-display-6;
+  }
+
+  p,
+  li,
+  .alert {
+    font-size: 1.063rem;
```

Comment:
Ahh I see this is where the 1.063 is coming from. Why isn't this just set to the body size?